frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -328,6 +330,17 @@ function ReaderHighlight:translate(selected_text)
 end
 
 function ReaderHighlight:onHoldRelease()
+    if self.hold_start_tv then
+        local hold_duration = TimeVal.now() - self.hold_start_tv
+        hold_duration = hold_duration.sec + hold_duration.usec/1000000
```

Comment:
TimeVal.sec seems to based on `math.floor(o.usec/1000000)` (rather than rounding) so as written this isn't necessary, but I suppose your point here is that it should be easy to switch to, say `2.5 s`.

https://github.com/koreader/koreader/blob/master/frontend/ui/timeval.lua#L16

Reply:
Well, I didnt think that much :) I already used that in TextBoxWidget:onHoldReleaseText(), where this duration is pass as a parameter to a callback, so there, rather be accurate so the callback get an accurate duration. I just cut & paster it here, and indeed, I could have shortcut that.
(And as TimeVal is used in gesturedetector, if it's good enough there, it's good enough here :)
Thanks for the merge!